python/opae.admin/opae/admin/tools/n5010tool.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+# Copyright(c) 2023, Silicom Denmark A/S
+#
+# Redistribution  and  use  in source  and  binary  forms,  with  or  without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of  source code  must retain the  above copyright notice,
+#   this list of conditions and the following disclaimer.
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+# * Neither the name  of Intel Corporation  nor the names of its contributors
+#   may be used to  endorse or promote  products derived  from this  software
+#   without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,  BUT NOT LIMITED TO,  THE
+# IMPLIED WARRANTIES OF  MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED.  IN NO EVENT  SHALL THE COPYRIGHT OWNER  OR CONTRIBUTORS BE
+# LIABLE  FOR  ANY  DIRECT,  INDIRECT,  INCIDENTAL,  SPECIAL,  EXEMPLARY,  OR
+# CONSEQUENTIAL  DAMAGES  (INCLUDING,  BUT  NOT LIMITED  TO,  PROCUREMENT  OF
+# SUBSTITUTE GOODS OR SERVICES;  LOSS OF USE,  DATA, OR PROFITS;  OR BUSINESS
+# INTERRUPTION)  HOWEVER CAUSED  AND ON ANY THEORY  OF LIABILITY,  WHETHER IN
+# CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.import opae.admin.tools.regmap_debugfs as regm
+
+import argparse
+import math
+import sys
+import re
+import os
+
+import opae.admin.tools.regmap_debugfs as regm
+from opae.admin.utils import max10_or_nios_version
+from opae.admin.fpga import fpga
+
+regmap_path = '/sys/kernel/debug/regmap/'
+
+N5010_BMC_OFFSET  = 0x300800
+N5010_QSFP0_RXPWR = 0x1c8
+N5010_PHY_CSR_0	  =	0x40c
```

Comment:
can you fix coding style?

pycodestyle n5010tool.py
n5010tool.py:40:17: E221 multiple spaces before operator
n5010tool.py:42:16: E223 tab before operator
n5010tool.py:42:20: E224 tab after operator
n5010tool.py:43:16: E223 tab before operator
n5010tool.py:43:20: E224 tab after operator
n5010tool.py:47:1: E302 expected 2 blank lines, found 1
n5010tool.py:57:1: E302 expected 2 blank lines, found 1
n5010tool.py:61:11: E271 multiple spaces after keyword
n5010tool.py:62:80: E501 line too long (107 > 79 characters)
n5010tool.py:70:1: E302 expected 2 blank lines, found 1
n5010tool.py:73:19: E211 whitespace before '('
n5010tool.py:73:22: E231 missing whitespace after ','
n5010tool.py:73:25: E231 missing whitespace after ','
n5010tool.py:78:23: E211 whitespace before '('
n5010tool.py:78:26: E231 missing whitespace after ','
n5010tool.py:85:1: E302 expected 2 blank lines, found 1
n5010tool.py:88:21: E231 missing whitespace after ','
n5010tool.py:129:1: E302 expected 2 blank lines, found 1
n5010tool.py:143:80: E501 line too long (115 > 79 characters)
n5010tool.py:147:1: E302 expected 2 blank lines, found 1
n5010tool.py:172:9: E265 block comment should start with '# '
n5010tool.py:174:80: E501 line too long (86 > 79 characters)
n5010tool.py:181:1: E305 expected 2 blank lines after class or function definition, found 1

Reply:
OK